src/dmd/json.d

```diff
@@ -289,6 +290,7 @@ public:
         case TRUST.system:  return property(name, "system");
         case TRUST.trusted: return property(name, "trusted");
         case TRUST.safe:    return property(name, "safe");
+        case TRUST.live:    return property(name, "live");
```

Comment:
?

Reply:
ditto

Reply:
The question is, why does astbase have a `live` member in the first place?  `@live` has nothing to do with `TRUST`.